deps/rabbitmq_mqtt/test/processor_SUITE.erl

```diff
@@ -31,38 +31,27 @@ suite() ->
 
 init_per_suite(Config) ->
     ok = application:load(rabbitmq_mqtt),
+    meck:new(rabbit_runtime_parameters, [passthrough, no_link]),
```

Comment:
If this mock is needed only for test case `get_vhost`, then this line should be moved into the `init_per_testcase` for that single test.